### ISSUE TITLE
Modifying a specify actor's trajectory 

### ISSUE THREAD

**Issue by bob020416** [wljungbergh/NeuroNCAP#10]
Hi, I'm really impressed by your works.

But i have a question how you perform actor modification through neurad, can you tell more how we can follow your step try to modify a car's trajectory  near ego ? 

Again, thanks for your great work

**Comment by rdesc** [comment]
I'm not one of the authors, but I found the information in the supplementary to be helpful https://www.ecva.net/papers/eccv_2024/papers_ECCV/papers/04335-supp.pdf

> We base our stationary scenarios on the following 10 NuScenes validation set sequences: 0099, 0101, 0103, 0106, 0108, 0278, 0331, 0783, 0796, 0966. For each sequence we pick 1-4 actors that have been observed from a sufficiently nearby distance and determine a collision location somewhere along the egotrajectory. We jitter this target position by 5-10m in the logitundinal direction; 0.5-1.5m in the lateral direction; and the yaw by 0.5-1.0 radians. Finally we determine a starting time of the scenario, so that a collision happens in slightly more than 4 seconds.

TLDR: The actor modifications are done manually by translating / rotating the logged trajectory such that a collision occurs with the ego

**Comment by rdesc** [comment]
Also see https://github.com/atonderski/neuro-ncap/issues/2

**Comment by bob020416** [comment]
Thank you so much, I have figured out a way to automatically modified safety critical scenario in a way, 

Now just wondering how do they deal with if the adverserial car rendering quality is bad on a side(since it is not being fully seen in the original nuscenes)

**Comment by rdesc** [comment]
Oh no way, cool! Can you share your code? Yea the poor rendering of parts of vehicles is hard to address
